CSharp/ArticleProjects/ApplicationInsights/Microsoft.Azure.Batch.Samples.TelemetryInitializer/ApplicationInsights.config

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Duplicate file? I guess maybe the one with the spelling mistake can just be deleted?